### DISCUSSION TITLE
New styling on the Documentation page is unreadable

### DISCUSSION THREAD

**Discussion by SavoInfostream** [adobe/spectrum-web-components#2792] [General]
Hey,
These new changes in styling of code elements on the Spectrum website are bad, IMO.
Colors are bad, and the syntax is new to me and not as useful as a normal HTML. 
Could you provide an option to toggle it of?
Thanks

![adobe](https://user-images.githubusercontent.com/60256229/206841463-46cddaac-8172-4b54-aed0-c21c4d5d54bb.png)

**Comment by Westbrook** [comment]
Thanks for the report! We're looking at addressing this oversight in https://github.com/adobe/spectrum-web-components/pull/2800
